bokehjs/test/unit/core/properties.ts

```diff
@@ -499,19 +499,19 @@ describe("properties module", () => {
       it("should default to data units", () => {
         const obj = new Some({distance_spec: {value: 10}})
         const prop = obj.properties.distance_spec
-        expect(prop.spec.units).to.be.equal("data")
+        expect(prop.units).to.be.equal("data")
```

Comment:
it would not surprise me if there are at least some people depending on this, at minimum needs a release note